cvcl/templates/assignment_detail.html

```diff
@@ -18,14 +18,35 @@
                 </a>
                 <button class="btn btn-primary" type="button" data-toggle="collapse" data-target="#collapseText"
                         aria-expanded="false" aria-controls="collapseText">
-                    {% bootstrap_icon "question-sign" %}Help
+                    {% bootstrap_icon "question-sign" %} Help
                 </button>
-                <div class="collapse" id="collapseText">
-                <div class="well">
-                    <h5>Lorem Ipsum</h5>
+                </div>
+
+                <div class="container-fluid">
+                    <div class="row">
+                        <div class="col-xs-12">
+                            <div class="collapse" id="collapseText">
+                                <div class="well well-sm">
+                                    {% if user.is_instructor %}
+                                        <h5><p>As an instructor, you will see a list of all Environments and their creators
```

Comment:
You can't put <p> inside <h5>. It's technically not a header, so remove <h5> completely. Apply this change to all files.